integration/auth_oidc_test.go

```diff
@@ -688,6 +689,108 @@ func TestOIDCReloginSameNodeNewUser(t *testing.T) {
 	}, 30*time.Second, 1*time.Second, "log out user2, and log into user1, no new node should be created")
 }
 
+func TestOIDCFollowUpUrl(t *testing.T) {
```

Comment:
This test looks great, I'm trying to improve the tests a bit with godoc that explains the flow of the test in a bit more human readable format, could you add that?